app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):
```

Comment:
Why you inherit from `ABC` you have no `@abstractmethods`

Reply:
Because pycharm raised problem (in atachment)
If it isn't necessary, i can write without this inheriting
![Снимок экрана 2022-08-15 в 18 50 00](https://user-images.githubusercontent.com/108237132/184679976-581ce5ed-3963-41eb-bd19-08ec2aecc6ed.png)